process/block/shardblock_test.go

```diff
@@ -1683,127 +1690,11 @@ func TestShardProcessor_CommitBlockStorageFailsForBodyShouldWork(t *testing.T) {
 	})
 
 	err = sp.CommitBlock(blkc, hdr, body)
-
+	wg.Wait()
 	assert.Nil(t, err)
 	assert.True(t, wasCalled)
 }
 
-func TestShardProcessor_CommitBlockNoTxInPoolShouldErr(t *testing.T) {
```

Comment:
why was this test removed?

Reply:
Because previously there was the below listed code in method CommitBlock from shard/meta processor and its return was treated wrongly as an critical error. At some point we have decided that all the errors received after a put action should be not critical and only logged with WARN.

Previous code from CommitBlock method:

err = mp.txCoordinator.SaveBlockDataToStorage(body)
if err != nil {
  return err
}

Now, with this refactor, I also corrected this part, which seems that remained as the only one treated differently. 

func (bp *baseProcessor) saveBody(body block.Body) {
	errNotCritical := bp.txCoordinator.SaveBlockDataToStorage(body)
	if errNotCritical != nil {
		log.Warn("saveBody.SaveBlockDataToStorage", "error", errNotCritical.Error())
	}
...
...
...
}